source/simulator/freesurface.cc

```diff
@@ -151,54 +163,39 @@ namespace aspect
 
 
   template <int dim>
-  void Simulator<dim>::FreeSurfaceHandler::execute()
+  void FreeSurfaceHandler<dim>::execute()
   {
     if (!sim.parameters.free_surface_enabled)
       return;
-    sim.computing_timer.enter_section("FreeSurface");
+    sim.computing_timer.enter_section("Free surface");
```

Comment:
ah, great. I like it much better that way. (I also always thought that the file name should be `free_surface.{h,cc}`, but that too is for a separate pull request.)